From 9566ebdb5003bc044393ef850c369d2447479629 Mon Sep 17 00:00:00 2001
Message-ID: <9566ebdb5003bc044393ef850c369d2447479629.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:41:02 +0100
Subject: [PATCH 10/19] gnu: go-github-com-jacobsa-ogletest: Move to (gnu
 packages golang-check).

* gnu/packages/golang-check.scm (go-github-com-jacobsa-ogletest): Move
from here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 36 +++++++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 35 ----------------------------------
 2 files changed, 36 insertions(+), 35 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index fe442bdb8b..f904e8b02c 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -122,6 +122,42 @@ (define-public go-github-com-jacobsa-oglemock
        "Package oglemock provides a mocking framework for unit tests.")
       (license license:asl2.0))))
 
+(define-public go-github-com-jacobsa-ogletest
+  (let ((commit "80d50a735a1108a2aeb7abc4a988d183f20c5292")
+        (revision "0"))
+    (package
+      (name "go-github-com-jacobsa-ogletest")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/jacobsa/ogletest")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "1lbwbxzr75g65q07ry5k4kglxqs3ym7xkvqznzm55rm3qk76v83r"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/jacobsa/ogletest"
+         ;; These tests should be made working
+         #:tests? #f))
+      (native-inputs
+       (list go-github-com-jacobsa-oglematchers
+             go-github-com-jacobsa-oglemock
+             go-github-com-jacobsa-reqtrace
+             go-golang-org-x-net))
+      (home-page "https://github.com/jacobsa/ogletest")
+      (synopsis "Expressive unit tests")
+      (description
+       "Package ogletest provides a framework for writing expressive unit tests.  It
+integrates with the builtin testing package, so it works with the gotest
+command.  Unlike the testing package which offers only basic capabilities for
+signalling failures, it offers ways to express expectations and get nice failure
+messages automatically.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-stretchr-testify
   (package
     (name "go-github-com-stretchr-testify")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index effe3e7854..04751b6a79 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1311,41 +1311,6 @@ (define-public go-github-com-aperturerobotics-jacobsa-crypto
 not included in the Go standard library.")
       (license license:asl2.0))))
 
-(define-public go-github-com-jacobsa-ogletest
-  (let ((commit "80d50a735a1108a2aeb7abc4a988d183f20c5292")
-        (revision "0"))
-    (package
-      (name "go-github-com-jacobsa-ogletest")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/jacobsa/ogletest")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "1lbwbxzr75g65q07ry5k4kglxqs3ym7xkvqznzm55rm3qk76v83r"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/jacobsa/ogletest"
-         ;; These tests should be made working
-         #:tests? #f))
-      (native-inputs (list
-                      go-github-com-jacobsa-oglematchers
-                      go-github-com-jacobsa-oglemock
-                      go-github-com-jacobsa-reqtrace
-                      go-golang-org-x-net))
-      (home-page "https://github.com/jacobsa/ogletest")
-      (synopsis "Expressive unit tests")
-      (description
-       "Package ogletest provides a framework for writing expressive unit tests.  It
-integrates with the builtin testing package, so it works with the gotest
-command.  Unlike the testing package which offers only basic capabilities for
-signalling failures, it offers ways to express expectations and get nice failure
-messages automatically.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-jacobsa-reqtrace
   (let ((commit "245c9e0234cb2ad542483a336324e982f1a22934")
         (revision "0"))
-- 
2.41.0

